<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gsm4</class>
 <widget class="QDialog" name="Gsm4">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="gbApn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>211</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="title">
    <string>DAFAULT APN</string>
   </property>
   <widget class="QRadioButton" name="rbCmnet">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>CMNET</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbMposcu">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>MPOSCUPSH.SH</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbVodafone">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>VODAFONE</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbGzylcmda">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>GZYLCMDA</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbGzdxcmda">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>GZDXCMDA</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbWxbc">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>WXBC</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbChinaUnicom">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>CHINA UNICOM</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbHide">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>HIDE</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="lTitle">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>GSM SETTING</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>224</width>
     <height>42</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pbCancel">
      <property name="minimumSize">
       <size>
        <width>90</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>CANCEL</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pbEnter">
      <property name="minimumSize">
       <size>
        <width>90</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>ENTER</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>211</y>
     <width>192</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>OTHER</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="leOther">
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>rbCmnet</tabstop>
  <tabstop>rbWxbc</tabstop>
  <tabstop>rbVodafone</tabstop>
  <tabstop>rbMposcu</tabstop>
  <tabstop>rbGzylcmda</tabstop>
  <tabstop>rbGzdxcmda</tabstop>
  <tabstop>rbChinaUnicom</tabstop>
  <tabstop>pbCancel</tabstop>
  <tabstop>pbEnter</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
